dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/HeaderFilter.java

```diff
@@ -22,5 +22,5 @@
 @SPI(scope = ExtensionScope.FRAMEWORK)
 public interface HeaderFilter {
 
-    RpcInvocation invoke(Invoker<?> invoker, RpcInvocation invocation) throws RpcException;
+    void invoke(Invoker<?> invoker, RpcInvocation invocation) throws RpcException;
```

Comment:
why change this api?

Reply:
+1

Reply:
I didn't notice that this API was introduced in 3.0. indeed shouldn't be modified, I'll revert it later. However, this return value not being used.

Reply:
Revert have done